skimage/transform/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""This module includes tools to transform images and volumetric data.
+"""Geometric and other transforms, e.g., rotation or the Radon transform.
```

Comment:
```suggestion
"""Geometric and other transformations, e.g., rotations, Radon transform.
```

Reply:
After a quick search it I'm still fuzzy on why this change is necessary. Seems to be that both terms are in use. But okay. :)

Reply:
When I suggest a change, I don't necessarily mean it's necessary; in this case, I just believe it's an improvement. :wink: 

I don't mean to say that "geometric transform" is incorrect, it's just that "geometric transformation" is the more canonical term. And the whole thing flows better, sort of avoiding a repetition: "(...) other transforms, (...) Radon transform". :relieved: